site/faq.md

```diff
@@ -99,6 +99,14 @@ explanation.
 Flux needs a deploy key to be allowed to push to the version control
 system in order to read and update the manifests.
 
+
+### What is auto-matically synced back to git by flux?
```

Comment:
Nothing is synced from the cluster to git -- syncing only goes one way, from git to the cluster.

There are some API calls (and `fluxctl` commands) that make updates to the files in git, and automated updates may also make changes in git, based on what you've told it to do. These go `command or automation -> git -> cluster`

Reply:
So it is potentially possible that in the future flux wouldn't even need write access to the git repo and would be able to operate in a limited mode without it?

Reply:
Yup; in theory (and some designs I've sketched out) it'd be possible to mark a repo as read only, so it will apply things to the cluster, but refuse updates.

Reply:
I've updated the PR. Is that better?